docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -61,15 +66,23 @@ To view, delete, or export existing lists:
 
 [float]
 [[detection-rule-exceptions]]
-=== Add detection exceptions to a rule
+=== Add exceptions to a rule
 
 You can add exceptions to a rule via the Rule details page or the Alerts table.
 When you add an exception, you can also close all alerts that meet the
 exception's criteria.
 
-IMPORTANT: When you select to close all alerts that meet the exception's
-criteria, all matching alerts are closed, *including* alerts generated by other
-rules.
+[IMPORTANT]
+==============
+Be careful when adding exceptions to EQL sequence rules. Exceptions are
+evaluated against every event in the sequence and when the exception matches a
+single event in the sequence, alerts are not generated. To exclude values from a
```

Comment:
```suggestion
evaluated against every event in the sequence and when the exception matches any
event(s) in the sequence, alerts are not generated. To exclude values from a
```

Just trying to disambiguate the behavior here; the use of "matches a single event" is less accurate than "matches any event(s)" IMO